tests/python/unittest/test_operator.py

```diff
@@ -4516,6 +4516,27 @@ def test_ctc_loss():
     true_loss = np.array([7.3557, 5.4091], dtype=np.float32) # from Torch
     check_ctc_loss(acts2, labels2, true_loss)
 
+    # Test 3: check use integer type as label
+    labels3 = np.array([[2, 3, 1], [2, 0, 0]], dtype=np.int32)
+    true_loss = np.array([7.3557, 5.4091], dtype=np.float32) # from Torch
+    check_ctc_loss(acts2, labels3, true_loss)
+
+@with_seed(1)
+def test_ctc_loss_with_large_classes():
+    ctx = default_context()
+    batch_size = 1024
+    seq_len = 35
+    label_len = 10
+    num_classes = 6000
+    x = np.random.uniform(size=(seq_len, batch_size, num_classes))
+    y = np.random.randint(0, num_classes, size=(batch_size, label_len))
```

Comment:
again this does not seem like a good way of testing this.

Reply:
Any suggestion to test the large classes? I could compare this with WarpCtc implementation result if that can be treated as golden.

Reply:
Make a small example, calculate a the value and test for that, like in any other CTC tests. Since this is for testing the type, the batch size and sequence lengths are irrelevant.

Reply:
The label type is tested in line 4520. This testcase is to test the large number of classes that would crash reported in issue #10995